group-work.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: episode
+title: How to organize a group's work
+teaching: 15
+exercises: 15
+questions:
+  - Should I start a repo under my account or open a new organization?
+  - Should I add everybody as collaborator?
+  - GitHub vs GitLab?
+  - What permissions to choose?
+  - Pull requests vs direct commits for types of projects
+  - Small vs large changes
+---
+
+# How to organize a group's work
+
+---
+
+## GitHub organizations
+
+Should I start a repository under my account or open a new organization?
+- Single-person projects often start under own account.
+- If is no problem to move a project from own namespace to an organization later.
+- When starting a larger project with several people, possibly several affiliations, an organization may be a better start.
+- If this is a GitHub pages project, then it will matter for the URL:
+  `myuser.github.io/myproject/` vs. `ourorg.github.io/ourproject/`.
+
+Should I add everybody as collaborator?
+- If you are a handful of project collaborators it probably makes sense to add everybody as collaborators.
+- But one does not have to be a *collaborator* to contribute (anybody can send contributions to public projects).
+- External contributors don't have to be added.
+
+---
+
+## Organizational permissions
+
+- Organizations have **owners** and **members**.
+- Owners can add more members and delete repositories.
+- Members can also be organized into teams.
+- We recommend to write-protect the default branch and protect it against force-pushes and accidental deletions.
+
+---
+
+## GitHub or GitLab?
+
+- GitHub: probably better integrations (with services like Zenodo), probably more visibility (more users).
+- GitLab: more features, you can also self-host, more advanced continuous integration.
+
+---
+
+## Direct commits or pull requests?
+
+- For single-person projects: direct pushes.
+- If you have somebody who can help you with code review: use pull requests.
+- For projects with 2 or more persons: agree on applying all changes via pull requests
+  and create a new branch for every change.
```

Comment:
One place to emphasize "if you have people who have enough time to pay attention to what others are doing."